src/pages/Login/LoginForm/EmailTextField.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { TextField } from '@mui/material'
+import { UseFormRegister, FieldErrors } from 'react-hook-form'
+
+import { LoginInputType } from './types'
+
+
+// eslint-disable-next-line no-useless-escape
+const emailRegex = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
```

Comment:
should be UPPER_CASE_SNAKE_CASE for constant

Reply:
renamed const to correct convention